scripts/data_files/driver_jsons/mbedtls_test_opaque_driver.json

```diff
@@ -0,0 +1,20 @@
+{
+    "prefix":       "mbedtls_test",
+    "type":         "opaque",
+    "location":     "0x7fffff",
+    "dependency":   "defined(PSA_CRYPTO_DRIVER_TEST)",
+    "headers":      ["test/drivers/test_driver.h"],
+    "capabilities": [
+        {
+            "_comment":     "The mbedTLS opaque driver supports import key/export key/export_public key",
+            "depend_on":    "defined(PSA_CRYPTO_DRIVER_TEST)",
+            "entry_points": ["import_key", "export_key", "export_public_key"]
+        },
+        {
+            "_comment":     "The mbedTLS opaque driver supports copy key/ get builtin key",
+            "depend_on":    "defined(PSA_CRYPTO_DRIVER_TEST)",
```

Comment:
`mbedtls/` namespace allowed in https://github.com/ARMmbed/mbedtls/pull/5512

Reply:
Technically this is more of a cosmetic differentiator and not a real global namespace we use. 
If we want a global namespace that's a lot more complicated, since that's really not global they are driver dependent.
Am assuming the cosmetic differentiator is exactly what you had in mind too !

Reply:
I don't understand what you mean by “cosmetic differentiator”. The goal is to differentiate properties that have the same meaning everywhere from properties that are only meaningful to Mbed TLS. `mbedtls/depends_on` would not be meaningful, for example, to a core written in Rust.

Reply:
Being cosmetic in its implementation, a user can understand looking at the name that this is a property meaningful only to Mbed TLS. However what it is not is 'a real namespace' wherein you create a namespace "mbedtls" and have specific properties around that.